zpay32/invoice_test.go

```diff
@@ -1,68 +1,12 @@
-package zpay32_test
+package zpay32
```

Comment:
Add a comment under the package name to explain why it is not `zpay32_test` (to share testdata with `internal–test`)